Sources/MultiSelectSegmentedControl.swift

```diff
@@ -17,6 +17,13 @@ import UIKit
 
     // MARK: - UISegmentedControl Enhancements
 
+    /// Optional selected background color, if different than tintColor
+    @objc public var selectedBackgroundColor: UIColor? {
```

Comment:
Forgot to mention:

Please make this an `@IBInspectable open dynamic var` To support Interface Builder and UIAppearance.

Thanks!

Reply:
Also, please move it to the end of the `Appearance` mark.
If you prefer, grant me [edit access to the PR](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) and I'll do these little fixes.)